src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -160,7 +160,8 @@ internal sealed class Imports
                             continue;
                         }
 
-                        var imported = usingsBinder.BindNamespaceOrTypeSymbol(usingDirective.Name, diagnostics, basesBeingResolved);
+                        var declarationBinder = usingsBinder.WithAdditionalFlags(BinderFlags.SuppressConstraintChecks);
+                        var imported = declarationBinder.BindNamespaceOrTypeSymbol(usingDirective.Name, diagnostics, basesBeingResolved);
```

Comment:
When do constraint checks on using`s run now that they no longer run during binding?